src/pages/tug/apply/Apply.component.js

```diff
@@ -0,0 +1,151 @@
+import React, { useState } from 'react';
+import Router from 'next/router';
+import { Modal, Button } from 'antd';
+
+import styles from './Apply.module.scss';
+import Container from '~/components/Container/Container';
+import { CoreLayout } from '~/layouts';
+import Form from './form';
+import { api } from '@tidb-community/datasource';
+
+const SuccessModal = (props) => (
+  // visible={isModalVisible} onOk={handleOk} onCancel={handleCancel}
+  <Modal title="提交成功！" {...props}>
+    <p>
+      我们将于 3 个工作日内对您提交的材料进行审核，并以【AskTUG 私信】形式告知您结果。
+      若审核通过，【微信小助手】将添加您为好友并邀请您进入 TUG 专属交流群，敬请留意。
+    </p>
+    <p>【AskTUG 私信】位置：社区导航栏 - 问答论坛 - 右上角头像下拉 - 私信</p>
+  </Modal>
+);
+
+const Apply = () => {
+  const title = 'TiDB User Group 会员申请';
```

Comment:
Split constant data into `apply.data.js`

Reply:
This content is only for testing, in the  production environment this content will be fetched from backend API